website/docs/enterprise/getting-started/install-enterprise.mdx

```diff
@@ -285,6 +1063,45 @@ WGE is now configured to ask users for authorization the next time a pull reques
 </TabItem>
 </Tabs>
 
+## TLS Configuration
+
+By default, the dashboard will listen on 0.0.0.0:9001 with TLS disabled and
```

Comment:
Don't really have a dashboard in EE.

This would be more the old

```
kubectl port-forward --namespace flux-system svc/clusters-service 8000:8000
```

Reply:
@foot I got the "by default..." language via the first sentence https://docs.gitops.weave.works/docs/configuration/tls/. Would the proper rewrite look something like:

"Use port-forwarding to connect to your Kubernetes cluster, using this command: `kubectl port-forward --namespace flux-system svc/clusters-service 8000:8000`"

Reply:
Yep, I can scribble some stuff in now

Reply:
Closing as I see your commit